app/helpers/textual_listview.rb

```diff
@@ -11,7 +11,12 @@ def locals
   end
 
   def self.new_from_hash(h)
-    new(*h.values_at(*members))
+    if h.nil?
+      temp = {:title => "Variables", :headers => ["Name", "Value"], :col_order => ["name", "value"], :value => []}
+      new(*temp.values_at(*members))
+    else
+      new(*h.values_at(*members))
+    end
```

Comment:
Simpler:

```suggestion
    h ||= {:title => "Variables", :headers => ["Name", "Value"], :col_order => ["name", "value"], :value => []}
    new(*h.values_at(*members))
```

Reply:
That being said, what if someone passes in an empty hash, or a hash missing something?

Reply:
Should I add a check to make sure each variable is present or did you have something else in mind?